drupal/manifests/drupal-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-secrets.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: Secret
+metadata:
+  name: drupal
+  labels:
```

Comment:
Useless for other actions, but it is useful for the action it covers.
I would like to get the secrets related to my drupal application instead of getting all secrets available in my cluster and manually select them, maybe I have a lot of secrets with random names from other helm chart sources or other kubernetes app deployments that will interfere with our awesome and standard secrets, easily filtered ;)